src/main/java/software/amazon/awssdk/crt/s3/S3FinishedResponseContext.java

```diff
@@ -53,6 +55,18 @@ public byte[] getErrorPayload() {
         return this.errorPayload;
     }
 
+    /**
+     * @return the name of the S3 operation that failed, in the case of a failed HTTP response.
+     * For example, if {@link S3MetaRequestOptions.MetaRequestType#PUT_OBJECT} fails
+     * this could be "PutObject", "CreateMultipartUpload", "UploadPart",
+     * "CompleteMultipartUpload", or others. For {@link S3MetaRequestOptions.MetaRequestType#DEFAULT},
+     * this is the name passed to {@link S3MetaRequestOptions#withOperationName}.
+     * May be null.
+     */
+    public String getErrorOperationName() {
+        return this.errorOperationName;
+    }
```

Comment:
Debatable: Can we always set it for success response as well? Might make things simpler. Ignore if this requires too much effort of changing it in the C code as well.

Reply:
Most of the members for S3FinishedResponseContext aren't set on success, since they're specifically about the properties of a failed HTTP response (headers, status-code, body, operation-name). And for something like multipart PUT_OBJECT, there are actually multiple different operations going on, so in the case of success ... which one would you be talking about?